tasks/setup-RedHat.yml

```diff
@@ -14,3 +14,9 @@
   package:
     name: logstash
     state: present
+
+# https://discuss.elastic.co/t/logstash-service-unit-not-found-centos-7/138446
+- name: Add Logstash systemd unit file
+  command:
+    cmd: "{{ logstash_dir }}/bin/system-install /etc/logstash/startup.options systemd"
+    creates: /etc/systemd/system/logstash.service
```

Comment:
If this _only_ affects CentOS 7, maybe we should put a `when` conditional on it to only have it run when the version == 7?